### ISSUE TITLE
clarification of requirements on copyright and license fields in media module

### ISSUE THREAD

**Issue by heathercole** [AAFC-BICoE/dina-planning#101]
GIVEN I have accessed DINA as data manager

WHEN media objects are added to the media module

THEN relevant copyright/license/attribution fields are filled in

AND these fields inform internal and external users about appropriate re-use

---
I had a good discussion with Stephanie Warner from the Office of Intellectual Property and Commercializing (OIPC).

Based on that conversation, there should be 3 fields used to describe the appropriate re-use of the digital assets
1. Copyright: [© Her Majesty the Queen in Right of Canada, represented by the Minister of Agriculture and Agri-Food] (text to be confirmed with OIPC)
2. License: Open Government Licence - Canada [with url: https://open.canada.ca/en/open-government-licence-canada]
3. Statement of Attribution/How to cite: [ declares copyright, license AND provided by [eg. which collection is source - text to be confirmed with OIPC] - this will better support collections if they want to try and track usage AND will clearly declare to users "how to cite"

From my perspective, i think the current "rights statement" field should be renamed to "Copyright", the "reuse rights statement" should be renamed to "Licence" and the text "Open Government Licence - Canada" should be used as well (instead) of the url currently listed.

Function:
when objects are uploaded, the default value for "copyright" is as above, as well as the License, and auto-complete of Statement of attribution (based on collection). This could also inform the default "publicly releasable" as "true". This should also be editable later in case there was an error

If uploader/editor changes the copyright to alternative (eg. "other"), then all related values should automatically be changes as relevant. License: "n/a", statement of attribution "n/a", publicly releasable = false.

**Comment by cgendreau** [comment]
The copyright should not change the license. It is perfectly valid to not own the copyright and have a licence like CC-BY (for a picture for example).

**Comment by heathercole** [comment]
I agree that is valid, however, if there are default values, and copyright is NOT "Crown/AAFC" then it would be important that a default of OGL-Canada is not maintained. I don't think it would be appropriate to have either field empty if the other held a value (unless CC-0). If licence is CC-BY, then there would have to be a Rights Holder/Copyright Identified/enforced, else both would have to be other/empty/n/a

Related, OIPC has replied regarding additional insight on the field names. I suggest no changes until I have confirmed details and relevant specific text.

**Comment by jmacklin** [comment]
A few comments here... The object being discussed is an image. That image has a copyright and an appropriate license. Attribution is always to the copyright holder but may be supplemented by additional metadata to designate a specific entity (person, institution...). The metadata associated with the image does not necessarily have the same licence. For the GoC it is very clear what the copyright is but not altogether clear what the licence is for certain object types. The OGL-C is slightly more restrictive than CC-BY and some entities will not be able to reuse these based on decisions made. For example, if we put an OGC-L licence on our DWC-As then GBIF will not take them (must be CC0 or CC-BY or CC-SA). Heather, it would be good to explore what OIPC says about using CC-BY on images as this would be preferable as it is a globally accepted licence. When I spoke with them and Treasury Board a few years ago about this issue they were okay with it.

As an aside, I also would argue that we should not be storing external images that are not openly sharable. This puts undue burden on the curators of the resource. What use cases support the need to do this? Even for research, I have difficulty defining any? There may be reasons not to share an image based on publication embargo or something like that but that is not related to it having a public licence. I think a policy that says only open images can be stored in DINA might be appropriate...?

Also note that for fields we are using Audubon Core which uses the global multimedia standards and I am certain TB would encourage/enforce use of these also. This is essential for interoperability and sharing. However, on the UI you have the freedom to call these whatever you like so that it makes sense for the user :-)

**Comment by heathercole** [comment]
I am following up with OIPC regarding appropriate terminology. In earlier discussions it was noted that while CC-BY (4.0) is an acceptable interpretation, that it would not be appropriate for AAFC to indicate any other licence than the OGL on its outward facing resources.

Collection Managers have identified the requirement of maintaining digital resources which are "not theirs to share" in support of managing collections. Therefore there is the requirement to support this use-case in the portal.

To my knowledge, no one has identified a use case where data associated with an image would have a different copyright or license than the image itself.

**Comment by cgendreau** [comment]
For images, not a problem.

For data, the reality is that OGL is not an acceptable license for sharing with GBIF: https://www.gbif.org/news/82812/licensing-milestone-for-data-access-in-gbiforg

**Comment by heathercole** [comment]
yes, for that purpose, the CC-BY IS an appropriate usage, but there is the difference between that usage, and stating it as the license.

Perhaps this difference needs to be more formally documented as it seems like 2 licenses would be possible depending on where the users were accessing it. Will review with OIPC. 

Related, i am reviewing the terms, and it seems like there is a combo of us using the darwin/audubon core terms vs. labels of terms. 
Normative URI	http://purl.org/dc/elements/1.1/rights
Modified	2020-01-27
Term version URI	http://dublincore.org/usage/terms/history/#rights-006
Label	Copyright Statement

**Comment by jmacklin** [comment]
"Collection Managers have identified the requirement of maintaining digital resources which are "not theirs to share" in support of managing collections. Therefore there is the requirement to support this use-case in the portal."

If the images and/or data is "not [ours] to share" then this is irrelevant to the portal/web presence as they will never appear there, right? If however we do have other institution's images and we have open permission to reuse/display them (this would typically be the case as the license dictates reuse) then there is no difference between making AAFC content accessible or anyone elses...

For the below, I am not exactly sure what the issue is but perhaps it is simply the difference of the term name associated with the data standard we are conforming to and whatever "label" we decide to put on our user interface to best describe what we want in a given field...? There is no conflict here. Also, the data standards are meant for interoperability and do not necessarily inform a particular schema or data model, so we do choose to map to many of them but there will be cases where we do not for various implementation reasons. In this case, as necessary we would transform the data as part of a process to be able to share it through say a DarwinCore Archive.

"Related, i am reviewing the terms, and it seems like there is a combo of us using the darwin/audubon core terms vs. labels of terms.
Normative URI http://purl.org/dc/elements/1.1/rights
Modified 2020-01-27
Term version URI http://dublincore.org/usage/terms/history/#rights-006
Label Copyright Statement"

**Comment by heathercole** [comment]
I have some answers from OIPC

**Attribution statement**: _(text to be provided for appropriate attribution as per both OGL-C and CC-BY licenses)_
"Images and data provided by [name of biological collection], ©Her Majesty The Queen in Right of Canada, as represented by the Minister of Agriculture and Agri-Food, licensed under the Open Government Licence – Canada (https://open.canada.ca/en/open-government-licence-canada)"
(i think the link could be embedded as a hyperlink, as the full-text provides enough information if the attribution was printed out) OR potentially hyperlink not required if there is a problem with that.

**Rights Holder**: _(OIPC has confirmed this is an appropriate label for this field not "Copyright" as I requested earlier)_
"Her Majesty The Queen in Right of Canada, as represented by the Minister of Agriculture and Agri-Food "
(if there is an issue with character limitations, we would have to review an acceptable abbreviation of some kind with OIPC)

Related, the "license" text should be text, not (only) the url, as the URL could change, but require the text to communicate the name of the license. "Open Government Licence - Canada"

I am still following up regarding OGL-C vs. CC-BY in the context of materials provided on the portal vs. harvested for aggregators like GBIF.

**Comment by heathercole** [comment]
regarding @jmacklin's comment above, everyone agrees that material that AAFC does not hold the rights to would NOT be shared on the portal. Using the owner/rights fields was a way that was suggested for managers to be able to make that distinction in the database. There may be other ways to support that requirement.

**Comment by cgendreau** [comment]
How do we handle bilingual text ?

**Comment by heathercole** [comment]
for bi-lingual, I think each of these fields would be set-text and have to be available in French and English for the portal. But I guess depends on how that is implemented in the portal. On this site https://www.agr.gc.ca/atlas/agclimate , clicking the 'about' section opens a pop-up with a copyright tab. Something similar may work, with a generic statement of attribution as a template with language toggle based on the website rather than the data.

I think necessary for portal side to support both, but presumably different ways to do it. 

Currently for data exports to the Canada Open Data portal, ALL field names/labels must be provided in french and english. I have some that I developed for a Type dataset I have been pushing through there, but is probably based on resources you know anyway.

**Comment by heathercole** [comment]
related to above, i think the same rights holder and license would apply to all portal content, only the attribution statement would vary, depending on the collection's data being accessed. That could be discussed, but was identified as a way that managers would be able to track/document usage/attribution to each collection.

**Comment by cgendreau** [comment]
Done
